src/client/components/Sorting/Sorting.component.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import './Sorting.styles.css';
+import { options } from './helper';
+import PropTypes from 'prop-types';
+
+const sortProducts = (sortMode, products) => {
+  if (sortMode === 'AlphabeticallyAZ') {
```

Comment:
its better to list all string option as const to use them in diferrent places and don`t repeat yourself 
const SORT_NAME_AZ = 'AlphbeticalAZ' etc..